packages/react-native-bridge/index.js

```diff
@@ -240,4 +240,11 @@ export function setStarterPageTemplatesTooltipShown( tooltipShown ) {
 	);
 }
 
+export function requestStoryCreatorLoad( mediaFiles, blockClientId ) {
+	return RNReactNativeGutenbergBridge.requestStoryCreatorLoad(
+		mediaFiles,
+		blockClientId
+	);
+}
```

Comment:
Stories are a WPCom-only feature, right?
If this is the case, it might not be good to reference it directly.

It might be better to have a more generic bridge method that could be used by the Story block or eventually other blocks with similar behavior.

Reply:
> It might be better to have a more generic bridge method that could be used by the Story block or eventually other blocks with similar behavior.

Excellent idea, here are the PRs tackling that (made them on top of the second part PRs to make the changes clearer)

Gutenberg: https://github.com/WordPress/gutenberg/pull/26005
Jetpack: https://github.com/Automattic/jetpack/pull/17456
Gutenberg mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2706
WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/13101